Deck_Shuffler/Deck_Shuffler/Deck.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deck_Shuffler
+{
+    class Deck
+    {
+        public List<Card> Cards = new List<Card>();
+        
+        public Deck()
+        {
+            string[] suits = new string[] { "Hearts", "Diamonds", "Clubs", "Spades" };
+            string[] values = new string[] { "Ace", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine", "Ten", "Jack", "Queen", "King" };
+            foreach (string suit in suits)
+            {
+                foreach (string value in values)
+                {
+                    Cards.Add(new Card(value, suit));
+                }
+            }
+        }
+        public Deck(string emptyDeck)
+        {
+            //Cards = null;
+        }
+
+        public void AddToDeck(string value, string suit)
```

Comment:
Again. Is this used? seems a bit strange after reading so far as it looks like the whole Desk is initialized in the constructor.

Reply:
Yes essentially what is going on in the shuffler is I take random cards out of EddsDeck and add them to an empty deck "ShuffledDeck" - ShuffledDeck begins life as the empty deck and uses AddToDeck to populate.